README.md

```diff
@@ -59,6 +59,19 @@ Two Recommendations are currently supported:
 - **Effective HPARecommend**: Recommend which workloads are suitable for autoscaling and provide optimized configurations such as minReplicas, maxReplicas.
 
 ### QoS Ensurance
+QoS Ensurance gaurantees the stability of the pods running on Kubernetes.
+Users can define PrioirtyClass for the pods and QoSEnsurancePolicy, and then crane detects disruption and ensure the high priority pods not being impacted by resource competition。
```

Comment:
suggest to change to the following statement.
`Kubernetes is capable of starting multiple pods on same node, and as a result, some of the user applications may be impacted when there are resources(e.g. cpu) consumption competition.
To mitigate this, Crane allows users defining PrioirtyClass for the pods and QoSEnsurancePolicy, and then detects disruption and ensure the high priority pods not being impacted by resource competition.`

Reply:
Modified it.